Examples/pixmaps.lisp

```diff
@@ -0,0 +1,172 @@
+;;;;  Copyright (c) 2020 Daniel Kochmański
+;;;;
+;;;;    License: BSD-2-Clause.
```

Comment:
Can this use the usual file header format? That format should work for any license.

Reply:
fixed